pkg/core/sentry/client.go

```diff
@@ -0,0 +1,41 @@
+package sentry
+
+import (
+	"os"
+
+	"github.com/getsentry/sentry-go"
+)
+
+var DefaultClient *Client
+
+func init() {
+	dsn := os.Getenv("SENTRY_DSN")
+	if dsn == "" {
+		DefaultClient = &Client{
+			Hub:   sentry.CurrentHub(),
+			Scope: sentry.CurrentHub().Scope(),
+		}
+		return
```

Comment:
If we did not provide SENTRY_DSN, how would the integration behave?

I saw we add the sentry hook unconditionally. And we always add the sentry middleware.

Reply:
If sentry is not configured, everything would still work, except the call to `CaptureEvent` would be a no-op.